### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by gongled**
Initial release

### CODE REVIEW

**Comment by savonarola** [comment]
No tests seem to be implemented; this is doubtedly suitable for production.

File: daemon/daemon.go [hunk 1]
```diff
@@ -0,0 +1,315 @@
+package daemon
```
Comment (savonarola) [inline daemon/daemon.go:1]: Does this tool really needs to be a daemon?
This requires many additional helper dependencies and many difficult to test code.

File: storage/cache/cache.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
+package storage
```
Comment (savonarola) [inline storage/cache/cache.go]: Is this module really used?

Comment (gongled) [reply storage/cache/cache.go]: I decided not to use cache functionality, but forgot about the orphaned code block. Removed (a1f91c03963991b609ec707272670b421413cdd0).

**Comment by gongled** [comment]
> No tests seem to be implemented; this is doubtedly suitable for production.

I am going to implement unit and integration tests soon.

**Pull request merged**